_guides/building-native-image-guide.adoc

```diff
@@ -25,13 +25,20 @@ Version {graalvm-version} is required.
 
 [NOTE]
 ====
-Once you have downloaded GraalVM, expand the archive and set the `GRAALVM_HOME` variable to this location:
+Once you have downloaded GraalVM, expand the archive and set the `GRAALVM_HOME` variable to its location:
 
 `export GRAALVM_HOME=$HOME/Development/graalvm/`
 
 On MacOS, point the variable to the `Home` sub-directory:
 
 `export GRAALVM_HOME=$HOME/Development/graalvm/Contents/Home/`
+
+You can also install GraalVM using https://sdkman.io/[sdkman].
+For example, to install the 1.0.0-RC12 version of GraalVM use:
+
+`sdk install java 1.0.0-rc-12-grl`. Then set the `GRAALVM_HOME` variable accordingly:
```

Comment:
And here is the trick... They use a different format for the version. So we need a second variable.